packages/composites/market-closure/test/e2e/checks.test.ts

```diff
@@ -1,66 +0,0 @@
-import { CheckProvider, getCheckImpl } from '../src/checks'
```

Comment:
Why is this file being removed?

Reply:
outdated e2e, they used methods that did not exist within the adapter, they practically did not test anything.